lock-api/src/main/java/com/palantir/lock/client/LockRefresher.java

```diff
@@ -29,7 +29,7 @@
 import com.palantir.lock.v2.LockToken;
 import com.palantir.lock.v2.TimelockService;
 
-public class LockRefresher implements AutoCloseable {
+public class LockRefresher {
```

Comment:
Isn't this an api break?

Reply:
depends when it gets merged... I added it in yesterday